lib/block/blocktransfer-error.js

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+'use strict';
+
+const errorCodes = require('./error-codes');
```

Comment:
IIUC, error-codes is used in both block and aem upload: https://github.com/adobe/node-httptransfer/blob/9ba21f111ad52e12080dd012f8a75fe29865db4a/lib/aem/aemupload.js#L34

so i think should `error-codes` file live somewhere not specific to block, wdyt?

Reply:
Why not. A suggestion where?

Reply:
I guess at the root of lib https://github.com/adobe/node-httptransfer/tree/master/lib